inaugurator/tests/test_server.py

```diff
@@ -4,6 +4,8 @@
 import time
 import subprocess
 import os
+import sys
+sys.path.insert(0, '.')
```

Comment:
this is considered bad. But other than that, also assumes a lot about how this file is going to be executed, regarding directories. my suggestion, if you want to follow through with your idea:
sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
import inaugurator
assert '/usr' not in inaugurator.__file__